gallery/src/components/NxModal/NxModalEscExample.tsx

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, {useState, useEffect} from 'react';
+
+import {NxModal, NxFontAwesomeIcon, NxButton, NxDropdown, NxP, useToggle} from '@sonatype/react-shared-components';
+import {faAngry} from '@fortawesome/free-solid-svg-icons';
+
+export default function NxModalStackedExample() {
+  const [showModal, setShowModal] = useState(false),
+      [showModal2, setShowModal2] = useState(false),
+      [showDropdown, toggleDropdown] = useToggle(false),
+      [showCustomComponent, setShowCustomComponent] = useState(false),
+      modal1CancelHandler = () => setShowModal(false),
+      modal2CancelHandler = () => setShowModal2(false);
+
+  useEffect(() => {
+    document.addEventListener('keydown', (evt) => {
+      if (evt.key === 'Escape' || evt.key === 'Esc') {
+        setShowCustomComponent(false);
+      }
+    });
+  }, []);
```

Comment:
Should we remove the `eventListener` on unmount?

Reply:
> Should we remove the eventListener on unmount?

Yes, thanks for the reminder. cb5dcebd